src/views/users/User/UserCreateUpdate.vue

```diff
@@ -116,6 +116,9 @@ export default {
         },
         system_roles: {
           component: Select2,
+          rules: [
+            rules.Required
+          ],
           el: {
             multiple: true,
             ajax: {
```

Comment:
There is no significant code difference detected between this version (2021-09-01) and your input. No issues or potential improvements have been identified in terms of coding style guidelines. If you require more detailed checks for different dates, please let me know!